test-clients/src/main/java/com/hedera/services/bdd/spec/util/DecodingUtil.java

```diff
@@ -0,0 +1,91 @@
+package com.hedera.services.bdd.spec.util;
+/*-
+ * ‌
+ * Hedera Services Node
+ * ​
+ * Copyright (C) 2018 - 2021 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import com.esaulpaugh.headlong.abi.Function;
+import com.esaulpaugh.headlong.abi.Tuple;
+import com.esaulpaugh.headlong.abi.TupleType;
+import com.google.common.primitives.Longs;
+import com.hedera.services.bdd.spec.HapiApiSpec;
+
+import java.util.Arrays;
+
+public class DecodingUtil {
+
+    static ResultType resultType = ResultType.NOT_SPECIFIED;
+
+    public static Tuple decodeResult(byte[] encodedReturn, String json) {
+
+        if (json.contains("address")) {
+            json = json.replaceAll("address", "bytes32");
```

Comment:
If we need to replace the `address`, it'd better be in the type returned from `function.getInputs(`). Replacing the `address` directly in the json, might have the side-effect of changing method names, e.g. `addressTransfer()` -> `bytes32Transfer()`